src/components/FeatureViews/Scheduler/SessionFilters.js

```diff
@@ -1,11 +1,12 @@
-import React, { useState } from "react";
+import React, {useState} from "react";
 import IconButton from "@material-ui/core/es/IconButton";
 import FilterIcon from "@material-ui/icons/FilterList";
 import Menu from "@material-ui/core/es/Menu";
-import { withStyles } from '@material-ui/core/styles';
+import {withStyles} from '@material-ui/core/styles';
 import blue from "@material-ui/core/es/colors/blue";
 import ReactSelect from "react-select";
-import { Tooltip, Typography, MenuItem } from "@material-ui/core";
+import MenuItem from "@material-ui/core/es/MenuItem";
+import Tooltip from "@material-ui/core/es/Tooltip";
```

Comment:
It is recommended to [avoid the `/es/` imports](https://material-ui.com/guides/minimizing-bundle-size/#ecmascript)